src/WorldStorage/WSSAnvil.cpp

```diff
@@ -3039,6 +3055,49 @@ void cWSSAnvil::LoadWolfOwner(cWolf & a_Wolf, const cParsedNBT & a_NBT, int a_Ta
 
 
 
+void cWSSAnvil::LoadOcelotOwner(cOcelot & a_Ocelot, const cParsedNBT & a_NBT, int a_TagIdx)
```

Comment:
Instead of duplicating the code, you should change the wolf function to something like 
```cpp
/** Loads the owner name and UUID from the entity at the specified NBT tag.
Returns a pair of {name, uuid}. If the entity is not owned, both are empty strings. */
std::pair<AString, AString> LoadEntityOwner(const cParsedNBT & a_NBT, int a_TagIdx);
```